src/main/java/com/kodilla/ecommercee/mapper/CartMapper.java

```diff
@@ -0,0 +1,33 @@
+package com.kodilla.ecommercee.mapper;
+
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.domain.dto.CartDto;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class CartMapper {
```

Comment:
Modify mapper to adapt to above comment review

Reply:
puściłem commita ze zmienionym mapper